doc/source/contributing.rst

```diff
@@ -626,6 +626,47 @@ This will display stderr from the benchmarks, and use your local
 Information on how to write a benchmark and how to use asv can be found in the
 `asv documentation <http://asv.readthedocs.org/en/latest/writing_benchmarks.html>`_.
 
+.. _contributing.gbq_integration_tests:
+
+Running Google BigQuery Integration Tests
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You will need to create a Google BigQuery private key in JSON format in
+order to run Google BigQuery integration tests on your local machine and
+on Travis-CI. The first step is to create a `service account
+<https://console.developers.google.com/iam-admin/serviceaccounts/>`__.
+
+Integration tests for ``pandas.io.gbq`` are skipped in pull requests because
+the credentials that are required for running Google BigQuery integration
+tests are `encrypted <https://docs.travis-ci.com/user/encrypting-files/>`__
+on Travis-CI and are only accessible from the pydata/pandas repository. In
+addition to creating a pull request on pydata/pandas, contributors should also
+create a pull request on their forked repository and add encrypted credentials
+to their fork. Here are the steps to run gbq integration tests on a forked
+repository:
+
+#. First, complete all the steps in the `Encrypting Files Prerequisites
+   <https://docs.travis-ci.com/user/encrypting-files/>`__ section.
+#. Sign into `Travis <https://travis-ci.org/>`__ using your GitHub account.
+#. Enable your forked repository of pandas for testing in `Travis
+   <https://travis-ci.org/profile/>`__.
+#. Run the following command from terminal where the current working directory
+   is the ``ci`` folder::
+
+    ./travis_encrypt_gbq.sh <your-gbq-json-credentials> <your-gbq-project-id>
+
+#. Commit the encrypted file called ``travis_gbq.json.enc`` as well as the
+   file ``travis_gbq_config.txt``, in an otherwise empty commit. DO NOT commit
+   the ``*.json`` file which contains your unencrypted private key.
```

Comment:
I've also removed my encrypted credentials from this branch, and created a new branch with my encrypted credentials.

https://github.com/pydata/pandas/compare/master...parthea:add-gbq-integration-test-setup-to-docs-my-credentials